tests/cacl/test_ebtables_application.py

```diff
@@ -16,7 +16,7 @@ def generate_expected_rules(duthost):
     return ebtables_rules
 
 
-def test_ebtables_application(duthosts, enum_rand_one_per_hwsku_hostname, enum_asic_index):
+def test_ebtables_application(request, duthosts, enum_rand_one_per_hwsku_hostname, enum_asic_index):
```

Comment:
This change seems at first glance to be unrelated. Is there a reason we need the `request` argument here?